packages/npm-dependencies-validation/src/utils/npmUtil.ts

```diff
@@ -27,9 +30,42 @@ export function invokeNPMCommand<T>(
   });
 }
 
+export function invokeNPMCommand(
+  config: NpmCommandConfig,
+  outputChannel: OutputChannel
+): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const command = executeSpawn(config, []);
+
+    // TODO: add start, cwd and end
+    command.stdout.on("data", (data) => {
+      sendDataToOutputChannel(`${data}`, outputChannel);
+    });
+
+    command.on("exit", () => {
+      resolve();
+    });
+
+    // TODO: does not stop here when an error occurs ?
+    command.on("error", (error) => {
+      const { stack } = error;
+      sendDataToOutputChannel(`${stack}`, outputChannel);
+      reject(error);
+    });
+  });
+}
+
+function executeSpawn(
```

Comment:
`executeSpawn` is a bad name because there is a similar API
to `spawn` in its own module (child-process) called `exec`

maybe `invokekSpawn`?